spec/requests/link_headers_spec.rb

```diff
@@ -13,7 +13,7 @@
     it 'contains webfinger url in link header' do
       link_header = link_header_with_type('application/jrd+json')
 
-      expect(link_header.href).to eq 'http://www.example.com/.well-known/webfinger?resource=acct%3Atest%40cb6e6126.ngrok.io'
+      expect(link_header.href).to eq 'http://cb6e6126.ngrok.io/.well-known/webfinger?resource=acct%3Atest%40cb6e6126.ngrok.io'
```

Comment:
There's a few more cases, but these seem to be setting the host when making the request:

https://github.com/search?q=repo%3Amastodon%2Fmastodon%20www.example.com&type=code